src/style.css

```diff
@@ -0,0 +1,26 @@
+body,
+ul,
+li {
+  margin: 0;
+  padding: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background: url(images/star.svg) no-repeat;
+  width: 16px;
+  height: 16px;
+  list-style-type: none;
+  margin: 0 2px;

```

Comment:
Remove it, and add margin-left: 4px. And your visual tests will be enter.

Reply:
It is'nt help me for visual test
I think problem in ul>li